src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -109,8 +109,8 @@
 
   <!-- Common dependencies across all targets -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.*" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.*" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.11.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.12.1" />
```

Comment:
For `Core`, let's use `2.11.*` instead of `2.11.0`. That makes it easier for us to automatically pick up patch releases as part of our build process.
For `AzureStorage`, we shouldn't need this change since the `1.12.*` should already pick up the latest patch release (which is `1.12.1`).

Reply:
We're breaking the build again with the 1.12.1 version change instead of 1.13.*. Can you update this to 1.13.*? I can upload this package to MyGet when you have it.

Reply:
Changed Core to 2.11.* and AzureStorage to 1.13.*.